pipeline_components/_tfdv/_tfdv/show_anomalies/component.yaml

```diff
@@ -0,0 +1,83 @@
+name: Show anomalies
+description: Analyzes an anomaly protobuf and reports the status.
+inputs:
+- {name: anomalies, type: Artifact, description: Input anomalies in protobuf text
+    format.}
+- name: fail_on_anomalies
+  type: Boolean
+  description: |-
+    Whether this step should fail if any anomalies are
+    found. Defaults to True.
+  default: "True"
+  optional: true
+- name: anomaly_code_ignore_list
+  type: JsonArray
+  description: |-
+    Optional list of codes of anomaly types to ignore as
+    anomalies. This list needs to contain the codes as integers &
+    NOT the anomaly types as strings.
+    Some examples of anomaly types include:
+    1. Code 19: FEATURE_TYPE_NOT_PRESENT - Column either not present
+       or has 100% nulls
+    2. Code 16: FEATURE_TYPE_LOW_FRACTION_PRESENT - % of non-null values in
+       a column is lower than expected
+    Reference for comprehensive list of anomaly types & their codes:
+    https://www.tensorflow.org/tfx/data_validation/anomalies
+    https://github.com/tensorflow/metadata/blob/master/tensorflow_metadata/proto/v0/anomalies.proto
+  optional: true
+implementation:
+  container:
+    image: python:3.7
+    command:
+    - sh
+    - -c
+    - |2
+
+      if ! [ -x "$(command -v pip)" ]; then
+          python3 -m ensurepip || python3 -m ensurepip --user || apt-get install python3-pip
+      fi
+
+      PIP_DISABLE_PIP_VERSION_CHECK=1 python3 -m pip install --quiet     --no-warn-script-location 'tensorflow-data-validation==1.6.0' 'kfp==1.8.9' && "$0" "$@"
+    - sh
+    - -ec
+    - |
+      program_path=$(mktemp -d)
+      printf "%s" "$0" > "$program_path/ephemeral_component.py"
+      python3 -m kfp.v2.components.executor_main                         --component_module_path                         "$program_path/ephemeral_component.py"                         "$@"
+    - "\nimport kfp\nfrom kfp.v2 import dsl\nfrom kfp.v2.dsl import *\nfrom typing\
```

Comment:
Is there a better way to store this for readability?

Reply:
remove component.yaml files